Engine/ac/room.cpp

```diff
@@ -603,12 +603,12 @@ void load_new_room(int newnum, CharacterInfo*forchar) {
         // the create_rgb_table call
         // so, fix them
         for (int ff = 0; ff < 256; ff++) {
-            if (palette[ff].r > 63)
-                palette[ff].r = 63;
-            if (palette[ff].g > 63)
-                palette[ff].g = 63;
-            if (palette[ff].b > 63)
-                palette[ff].b = 63;
+            if (palette[ff].r > 255)
+                palette[ff].r = 255;
+            if (palette[ff].g > 255)
+                palette[ff].g = 255;
+            if (palette[ff].b > 255)
+                palette[ff].b = 255;
```

Comment:
This code chunk may be removed completely, because palette components are declared as `unsigned char` and can never be > 255.